test/Table_Tests/src/In_Memory/Column_Format_Spec.enso

```diff
@@ -86,25 +89,35 @@ spec =
 
         Test.specify "Date_Time with locale" <|
             input = Column.from_vector "values" [Date_Time.new 2020 6 21 8 10 20, Date_Time.new 2023 4 25 14 25 2]
-            expected_default = Column.from_vector "values" ["21. June 2020 08.10.20", "25. April 2023 14.25.02"]
-            expected_gb = Column.from_vector "values" ["21. Jun 2020 08.10.20", "25. Apr 2023 14.25.02"]
+            expected_default = Column.from_vector "values" ["21. Jun 2020 08.10.20", "25. Apr 2023 14.25.02"]
+            expected_gb = Column.from_vector "values" ["21. June 2020 08.10.20", "25. April 2023 14.25.02"]
             expected_fr = Column.from_vector "values" ["21. juin 2020 08.10.20", "25. avril 2023 14.25.02"]
             input.format "d. MMMM yyyy HH.mm.ss" . should_equal expected_default
-            input.format "d. MMMM yyyy HH.mm.ss" (Locale.default) . should_equal expected_default
-            input.format "d. MMMM yyyy HH.mm.ss" (Locale.new "gb") . should_equal expected_gb
-            input.format "d. MMMM yyyy HH.mm.ss" (Locale.new "fr") . should_equal expected_fr
+            input.format (Date_Time_Formatter.from "d. MMMM yyyy HH.mm.ss" Locale.default) . should_equal expected_default
+            input.format (Date_Time_Formatter.from "d. MMMM yyyy HH.mm.ss" Locale.uk) . should_equal expected_gb
+            input.format (Date_Time_Formatter.from "d. MMMM yyyy HH.mm.ss" Locale.france) . should_equal expected_fr
+
+        Test.specify "overriding the Locale with `format` argument" <|
+            formatter = Date_Time_Formatter.from "d. MMMM yyyy HH.mm.ss" Locale.france
+            input = Column.from_vector "values" [Date_Time.new 2020 6 21 8 10 20, Date_Time.new 2023 4 25 14 25 2]
+            expected_fr = Column.from_vector "values" ["21. juin 2020 08.10.20", "25. avril 2023 14.25.02"]
+            expected_pl = Column.from_vector "values" ["21. czerwca 2020 08.10.20", "25. kwietnia 2023 14.25.02"]
+
+            input.format formatter . should_equal expected_fr
+            # If I provide a locale argument, it overrides what is already in the formatter:
+            input.format formatter Locale.poland . should_equal expected_pl
 
         Test.specify "Empty/Nothing format" <|
             zone = Time_Zone.parse "US/Hawaii"
             input = Column.from_vector "values" [Date_Time.new 2020 12 21 8 10 20 zone=zone, Date_Time.new 2023 4 25 14 25 2 zone=zone]
-            expected = Column.from_vector "values" ['2020-12-21T08:10:20-10:00[US/Hawaii]', '2023-04-25T14:25:02-10:00[US/Hawaii]']
```

Comment:
I decided to remove the `T` from our default format - it seems that this is the format we prefer. It is not exactly ISO though.

@jdunkerley do you think it's a good change or shall I undo this?

Reply:
They both are working for _read_. But we need to pick one for Output, so I chosen the form without `T`. When parsing both `T` and ` ` will work.

Reply:
ah ok - that was the old form too (that was why we had an "output formatter") I thought.

Reply:
Hm, I thought it wasn't because after all I _did have to_ change these tests. Maybe we were not using output formatter in all the places we should. Anyway - now it should be uniform I think.